Lib/fontbakery/profiles/googlefonts_conditions.py

```diff
@@ -16,10 +16,21 @@
 def style(font):
     """Determine font style from canonical filename."""
     from fontbakery.constants import STATIC_STYLE_NAMES
+    acceptable_stylenames = [name.replace(' ', '') for name in STATIC_STYLE_NAMES]
     filename = os.path.basename(font)
-    if '-' in filename:
+    # VF
+    if '[' in filename:
+        filename = os.path.splitext(filename)[0].split('[')[0]
+        if '-' in filename:
+            stylename = os.path.splitext(filename)[0].split('-')[1]
```

Comment:
you don't need to `os.path.splitext()[0]` a second time here, as you're already dealing with the result of that operation that was stored on the `filename` variable.